v2/pkg/engine/resolve/resolve_test.go

```diff
@@ -5062,6 +5072,35 @@ func TestResolver_ResolveGraphQLSubscription(t *testing.T) {
 		recorder.AwaitComplete(t, defaultTimeout)
 		fakeStream.AwaitIsDone(t, defaultTimeout)
 	})
+
+	t.Run("renders query plan with trigger", func(t *testing.T) {
```

Comment:
Please add a test that contains both a trigger and at least one additional fetch.

Reply:
Done